2008-09-29  Michael Tautschnig  <mt@debian.org>

	* bin/setup-storage: Enable debug mode if -d was given on the command line
Index: trunk/bin/setup-storage
===================================================================
--- trunk.orig/bin/setup-storage
+++ trunk/bin/setup-storage	
@@ -89,6 +89,9 @@
 use Fstab;
 use Exec;
 
+# enable debug mode, if requested using -d
+$opt_d and $FAI::debug = 1;
+
 # the config source file
 my $config_file = undef;
 # use the config file, if given
